deploy/playbooks/roles/common/handlers/main.yml

```diff
@@ -6,19 +6,16 @@
   command: systemctl daemon-reload
 
 - name: restart app
-  register: app_restart
   sudo: yes
   service: name=chacra state=restarted enabled=yes
 
-- name: restart chacra-celery
+- name: restart celery
```

Comment:
can you keep `chacra-celery` ? it is better to have a reminder of the actual service that is being restarted. I've been confused before by seeing "restarting celery" but it is in fact restarting "chacra-celery"